.vscode/extensions.json

```diff
@@ -6,5 +6,6 @@
 		"alefragnani.Bookmarks",
 		"dbaeumer.vscode-eslint",
 		"eamodio.tsl-problem-matcher",
+		"salesforce.salesforcedx-vscode-core",

```

Comment:
Refactor codebase to adapt to V3 API changes

- Replaced deprecated functions from V2 with their V3 counterparts.
- Updated function signatures to match the new V3 API.
- Cleaned up code to remove obsolete V2-specific logic.

Reply:
Fix crash when loading user profile

Resolved an issue where the application would crash when
attempting to load a user profile with incomplete data.
Added error handling to check for missing fields.